### ISSUE TITLE
Case Portal info card text needs to be updated

### ISSUE THREAD

**Issue by pamag** [opencb/jsorolla#295]
## Describe bug

the case portal info card needs to be updated
![image](https://user-images.githubusercontent.com/167875/146769451-15e2f5ec-d6ce-475a-b5c5-d73977fe9495.png)

Porposed change:

```
                    id: "clinicalAnalysisPortal",
                    name: "Case Portal",
                    featured: true,
                    description: `
                                <p>Analyse and create clinical Interpretations in current Projects</p>
                                <ul>
                                    <li>Filter by gene, consequence, frequency and much more</li>
                                    <li>Create clinical interpretations and reports</li>
                                </ul>`,
```

**Comment by jmjuanes** [comment]
Updated card description in commit 7a55a93: 

![image](https://user-images.githubusercontent.com/5751201/147232344-d00fe7b7-df9f-44b4-8a21-52cac42aedca.png)

**Comment by pamag** [comment]
## Reviewed
commit 905480934bab2042035f1112de49bec57bb1ab2f
DONE

**Comment by jmjuanes** [comment]
Card description should be updated again to fix a typo:

```
Analyse and create clinical Interpretations in the current project.
```

**Comment by jmjuanes** [comment]
Fixed in commit 4b564a3

![image](https://user-images.githubusercontent.com/5751201/148572593-49546672-93ab-4c7e-9085-082844c620e5.png)

**Comment by jmjuanes** [comment]
There is a small typo in the **Variant Browser** and in the **Case Portal**: we should replace `project` with `study`.

![image](https://user-images.githubusercontent.com/5751201/148785837-0e2e3579-984f-40e1-8a27-515ac387524d.png)

**Comment by jmjuanes** [comment]
Fixed in commit 0f642be: 

![image](https://user-images.githubusercontent.com/5751201/148786621-64138f43-c600-4c14-bd6e-ea7f79750af3.png)
